client/src/components/ModuleDetails/ModuleDetails.js

```diff
@@ -49,7 +49,7 @@ class ModuleDetails extends Component {
           {
             securityAlerts.map((alert) => (
               <li key={alert.Identifier}>
-                <a href={alert.ExternalLink} target="_blank" rel="noopener">
+                <a href={alert.ExternalLink} target="_blank" rel="noreferrer">
```

Comment:
What does that do?

Reply:
Security thing, there was an NPM warning about saying to add `noreferrer`, which does what `noopener` does + a bit more

https://web.dev/external-anchors-use-rel-noopener/